py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -296,13 +298,25 @@ def create_trt_exp_program(
     # torch.export serialization expects them to be lifted
     gm, trt_graph_signature, state_dict = lift(gm, trt_graph_signature)
 
+    module_call_graph = [
+        ModuleCallEntry(
+            "",
+            ModuleCallSignature(
+                inputs=[],
+                outputs=[],
```

Comment:
Are the `ArgumentSpec` specifications not needed here?

Reply:
yep not required apparently.